src/js/flagset.ts

```diff
@@ -1,524 +1,76 @@
+import {DEBUG_MODE_FLAGS} from './flags/debug-mode';
```

Comment:
FYI: the import files need to be *.js - it's a bit weird, but it's required for my setup and TS handles it fine.